web/src/lib/components/album-page/album-card.svelte

```diff
@@ -61,35 +63,39 @@
 </script>
 
 <div
-	class="hover:cursor-pointer mt-4 relative"
+	class="hover:cursor-pointer mt-4 border-2 border-transparent dark:hover:border-gray-300 hover:border-black rounded-xl p-5"
```

Comment:
What do you think about using `rounded-3xl`?

![image](https://github.com/immich-app/immich/assets/59014050/1f9d2305-f22e-4668-b63b-ad8a4dd129a3)![image](https://github.com/immich-app/immich/assets/59014050/f80f272e-ae29-43f6-b731-d2596e128a9e)